kratos/utilities/force_and_torque_utils.cpp

```diff
@@ -0,0 +1,75 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+
+/* System includes */
+
+/* External includes */
+
+/* Project includes */
+#include "utilities/force_and_torque_utils.h"
+#include "utilities/parallel_utilities.h"
+#include "utilities/reduction_utilities.h"
+#include "utilities/math_utils.h"
+
+namespace Kratos
+{
+
+
+std::array<array_1d<double,3>,2> ForceAndTorqueUtils::SumForceAndTorque(const ModelPart& rModelPart,
+                                                                        const array_1d<double,3>& rReferencePoint)
```

Comment:
```suggestion
std::array<array_1d<double,3>,2> ForceAndTorqueUtils::SumForceAndTorque(
    const ModelPart& rModelPart,
    const array_1d<double,3>& rReferencePoint)
```
Style guide. 

I also think we should ask the user for the variable to be used.

Reply:
Indeed, I think we should have a `SumForce` which only asks for the force variable and a `SumForceAndTorque` which asks for both the force and moment variables.

Reply:
makes sense to me!